src/Schema/Events/Conversations.ts

```diff
@@ -0,0 +1,48 @@
+import { ActionType } from "../Event"
+
+/**
+ * Schemas describing Conversation events
+ * @packageDocumentation
+ */
+
+ /**
+ * A user focused on message input on a conversation
+ *
+ * This schema describes events sent to Segment from [[focusedOnConversationMessageInput]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "focusedOnConversationMessageInput",
+ *    impulse_conversation_id: 198
+ *    
+ *  }
+ * ```
+ */
+export interface FocusedOnConversationMessageInput {
+    action: ActionType.focusedOnConversationMessageInput
+    impulse_conversation_id?: number
+  }
+
+/**
+ * A user successfully sends a message to partner from a conversation
+ *
+ * This schema describes events sent to Segment from [[sentConversationMessage]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "sentConversationMessage",
+ *    impulse_conversation_id: 198,
+ *    impulse_message_id: 8765
+ *    
+ *  }
+ * ```
+ */
+export interface SentConversationMessage {
+    action: ActionType.sentConversationMessage
+    impulse_conversation_id?: number
```

Comment:
Is there an instance where a conversation or message does not have an `id`?

Reply:
it should actually always have it, so i can remove the `?`!
also, i am resolving the conflicts but getting a yarn error, anyone available for a quick pair?

Reply:
nvm fixed it!

Reply:
i do also wonder if `number` is the right type given that these are postgresql ids?

Reply:
If redshift receives them as `number` then this type makes sense. And yes, the `?` being removed will make the property required.